changelogs/fragments/772-Encode-files-recursively-and-test-case-for-keep-behavior.yml

```diff
@@ -0,0 +1,6 @@
+bugfixes:
+  - zos_copy - Zos_copy did not encode inner content of a inner folder on 
```

Comment:
Probably nitpicking here, but the fragment should start with `zos_copy` instead of `Zos_copy`.

I'd also change `of a inner folder on the source is copy to the destiny` to `inside subdirectories once the source was copied to the destination`